dockyard/controllers/v1/container.py

```diff
@@ -0,0 +1,210 @@
+from pecan import expose, abort, request
+
+from dockyard.common.container import container 
+
+class Container(object):
+    def __init__(self):
+        self.container = container.Container()
+
+    @expose(generic=True)
+    def index(self):
+        abort(404)
+
+    @index.when(method="DELETE")
+    def delete(self, _id, **kwargs):
+        return self.container.delete(_id, **kwargs)
+
+    @expose(generic=True)
+    def resize(self, _id, **kwargs):
+        abort(404)
+
+    @resize.when(method="POST")
+    def resize_(self, _id, **kwargs):
+        return self.container.resize(_id, **kwargs)
+
+    @expose()
+    def changes(self, _id):
+        return self.container.changes(_id)
+
+    @expose()
+    def export(self, _id):
+        return self.container.export(_id)
+
+    @expose()
+    def json(self, name_id=None):
+        query_params = request.query_string
+        return self.container.list(name_id,  query_params=query_params)
+
+    @expose()
+    def stats(self, _id):
```

Comment:
It does not work because stat returns streams therefor we have to find out a method where streams can be redirected